packages/oas-resolver/index.js

```diff
@@ -18,25 +18,22 @@ function unique(arr) {
     return [... new Set(arr)];
 }
 
-function readFileAsync(filename, encoding, files = null) {
+function readFileAsync(filename, encoding, options, pointer, def) {
     return new Promise(function (resolve, reject) {
-        filename = decodeURI(filename);
```

Comment:
This PR is overridding all the changes done to get the oas resolver work on browser. 
@VShingala

Reply:
We have to pass the files object to this function to read the file from the path <> blob map in case of browser.

Reply:
@umeshp7  Have updated PR to contain changes made by you to work on browser.

Reply:
We will also need to add the handling for `options.ignoreIOErrors && def` when files maps is passed.

Reply:
Yes Actually, adding them too. Will need/introduce that option in multifile import too.

Reply:
What do you mean introduce option to multifile import? This is an oas-resolver specific option right? Or are we exposing it through the import options as well? Is yes, why is it needed? Can't we just ignoreIOErrors automatically?

Reply:
I meant we use resolver for multifile schema resolution so same error is also present there where one invalid ref will throw error instead not resolving it. Hence using same option in that flow too. Not needed in this module but was talking of making change in `openapi-to-postman`.

Reply:
There is only one place from where we call the oas resolver adding the option there should suffice.